docs/technical_reference/unit_models/reverse_osmosis_0D.rst

```diff
@@ -159,6 +160,7 @@ Equations
    "Solvent flux across membrane (SKK)", ":math:`J_{solvent} = \rho_{solvent} A(P_{f} - P_p - \sigma(\pi_{f}-\pi_{p}))`"
    "Solute flux across membrane (solution-diffusion)", ":math:`J_{solute} = B(C_{f} - C_{p})`"
    "Solute flux across membrane (SKK)", ":math:`J_{solute} = B(C_{f} - C_{p}) + (1 - \sigma)\frac{J_{solvent}}{\rho_{solvent}}C_{f}`"
+   "Alpha (SKK)", ":math:`\alpha = \frac{1 - \sigma}{B}`" 
```

Comment:
@fahim831 I moved alpha here just to have it follow other rows having to do with the Spiegler-Kedem-Katchalskiy model (SKK).